src/modules/auth/auth.service.ts

```diff
@@ -44,10 +44,16 @@ export class AuthService {
     const user = this.accountRepository.create({
       email,
       password: hashedPassword,
-      ...rest,
     });
     await this.accountRepository.save(user);
 
+    const userProfile = this.profileRepository.create({
+      accountId: { id: user.id },
```

Comment:
This doesn't seem right. To fix it, I would ask you to fix the profile entity file. Currently, ```accountId: Account``` exists there. Instead, we need ```account: Account``` and ```accountId: string```.

Reply:
Fixed